system/betterlockscreen@.service

```diff
@@ -1,12 +1,15 @@
 [Unit]
 Description = Lock screen when going to sleep/suspend
+Before=sleep.target
+Before=suspend.target
```

Comment:
Should we include hibernate.target hybrid-sleep.target to support all types?

Reply:
Sounds reasonable. I don't use this, so I don't really care either way.